.github/workflows/post-pr.yml

```diff
@@ -10,7 +10,7 @@ on:
 
 jobs:
   noop:
-    runs-on: ubuntu-latest
```

Comment:
I would keep `-latest` for this, because it will never need an update that way.

Reply:
Please revert this one.

Reply:
Done in [49a2603](https://github.com/ECP-WarpX/WarpX/pull/5731/commits/49a260316d6714dfba254b0eeea00a6f11833641).